sydent/http/servlets/replication.py

```diff
@@ -247,7 +247,7 @@ def _async_render_POST(self, request):
         # Then extract the list of invite token update dictionaries, ensuring
         # tokens are processed in order of origin_id
         invite_updates = sorted(
-            invite_updates, key=lambda k: k["origin_id"]
+            invite_updates, key=lambda k: int(k["origin_id"])
```

Comment:
Are these not already ints?

Reply:
Maaaaaybe. I need to figure out exactly what sqlite returns, which I'll do when I get around testing the replication of invite updates. In the meantime, I think it safer to go with "overly safe" rather than "maybe already safe but who knows".